bitnami/superset/README.md

```diff
@@ -0,0 +1,750 @@
+<!--- app-name: superset -->
+
+# Bitnami package for Apache Superset
+
+Superset is an open source Data Visualization and Data Exploration Platform.
+
+[Overview of Apache Superset](https://superset.apache.org/)
+
+Trademarks: This software listing is packaged by Bitnami. The respective trademarks mentioned in the offering are owned by the respective companies, and use of them does not imply any affiliation or endorsement.
+
+## TL;DR
+
+```console
+helm install my-release oci://registry-1.docker.io/bitnamicharts/superset
+```
+
+Looking to use Superset in production? Try [VMware Tanzu Application Catalog](https://bitnami.com/enterprise), the enterprise edition of Bitnami Application Catalog.
+
+## Introduction
+
+Bitnami charts for Helm are carefully engineered, actively maintained and are the quickest and easiest way to deploy containers on a Kubernetes cluster that are ready to handle production workloads.
+
+This chart bootstraps a [Superset](https://superset.apache.org/) Deployment in a [Kubernetes](https://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+Bitnami charts can be used with [Kubeapps](https://kubeapps.dev/) for deployment and management of Helm Charts in clusters.
+
+## Prerequisites
+
+- Kubernetes 1.23+
+- Helm 3.8.0+
+- PV provisioner support in the underlying infrastructure
+- ReadWriteMany volumes for deployment scaling
```

Comment:
This chart doesn't have a "persistence" section and the sub-charts (Redis and PostgreSQL) use StatefulSets with PVC Claim Templates, hence I don't think RWX is a requirement:
```suggestion
```